packages/utils/src/misc.ts

```diff
@@ -369,12 +369,17 @@ export async function handleDeeplinkRedirect({
 
     const json = typeof wcDeepLink === "string" ? JSON.parse(wcDeepLink) : wcDeepLink;
     let deeplink = json?.href;
-
     if (typeof deeplink !== "string") return;
 
+    const payload = `requestId=${id}&sessionTopic=${topic}`;
     if (deeplink.endsWith("/")) deeplink = deeplink.slice(0, -1);
-
-    const link = `${deeplink}/wc?requestId=${id}&sessionTopic=${topic}`;
+    let link = `${deeplink}`;
```

Comment:
Curious why do we need to interpolate this ?
```suggestion
    let link = deeplink;
```

Reply:
leftover from previous behaviour